packages/search-adapter/src/empathy/requestors/beacon-tracking.requestor.ts

```diff
@@ -2,27 +2,51 @@ import { injectable } from 'inversify';
 import { TrackingRequest } from '../../types';
 import { Requestor } from '../empathy-adapter.types';
 
+let isAdblockPresent = false;
+detectAdblock()
+  .then(() => (isAdblockPresent = false))
+  .catch(() => (isAdblockPresent = true));
+
 /**
  * TODO https://searchbroker.atlassian.net/browse/EX-2163
```

Comment:
This links nowhere, what was the TODO about?

Reply:
This was about documenting the entire adapter :rip: